package.json

```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-prettier": "^3.1.3",
     "husky": "^4.2.5",
     "jest": "^26.0.1",
+    "metro-react-native-babel-preset": "^0.66.2",
```

Comment:
The babel transform that was happening during the jest run needed this preset, but it wasn't there.  Explicitly adding it as a dev dependency here.